linkernetworks/azure-intelligent-edge-patterns
# Fix/stream cannot open http

## Purpose
* Fix rstp open http error

## Does this introduce a breaking change?
```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
pytest vision_on_edge/streams
pytest vision_on_edge/cameras
```

* Test the code
Open camera, input http
check
1. stream live view ok
2. capture ok

## What to Check
* ...

## Other Information
